site/app/templates/grading/electronic/Status.twig

```diff
@@ -176,13 +39,568 @@
+                                </div>
+                            {% endif %}
+                            <br/>
+                            <b>Statistics for Manually Graded Components: </b>
+                            <br/>
+                            <div style="margin-left: 20px">
+                                {% if component_averages|length == 0 %}
+                                    No components have been graded yet.
+                                {% else %}
+                                    {% for component in component_averages %}
+                                        {{ component.getTitle() }}:<br/>
+                                        <div style="margin-left: 40px">
+                                            Average:
+                                            {% if component.getMaxValue() > 0 %}
+                                                {{ component.getAverageScore() }} / {{ component.getMaxValue() }} ({{ component_percentages[loop.index0] }}%)
+                                            {% else %}
+                                                {{ component.getAverageScore() }}
+                                            {% endif %}
+                                            <br/>
+                                            Standard Deviation: {{ component.getStandardDeviation() }} <br/>
+                                            Count: {{ component.getCount() }} <br/>
+                                        </div>
+                                    {% endfor %}
+                                    <br/>
+                                    Overall Average: {{ component_overall_score }} / {{ component_overall_max }} ({{ component_overall_percentage }}%)
+                                {% endif %}
+                            </div>
+                        {% endif %}
+                        {# RIGHT CHUNK #}
+                    </div>
+
+
+                </div>
+
+            </div>
+        </div>
+    {% endif %}
 </div>
+</body>
+{# Tab Buttons #}
+<script>
+    function openStat(evt, statName) {
+        // Declare all variables
+        var i, tabcontent, tablinks;
+
+        // Get all elements with class="tabcontent" and hide them
+        tabcontent = document.getElementsByClassName("tabcontent");
+        for (i = 0; i < tabcontent.length; i++) {
+            tabcontent[i].style.display = "none";
+        }
+
+        // Get all elements with class="tablinks" and remove the class "active"
+        tablinks = document.getElementsByClassName("tablinks");
+        for (i = 0; i < tablinks.length; i++) {
+            tablinks[i].className = tablinks[i].className.replace(" active", "");
+        }
+
+        // Show the current tab, and add an "active" class to the button that opened the tab
+        document.getElementById(statName).style.display = "block";
+        //to make active, do this, not good for current implementation
+        //evt.currentTarget.className += " active";
```

Comment:
I do feel there needs to be some sort of mechanism to tell me which page I am in though... What do you think?

Reply:
![fixedtabs 2019-06-18 10-36](https://user-images.githubusercontent.com/36307579/59693596-2f4a0900-91b5-11e9-8ee3-abe6eb735c3f.gif)